virt-v2v/cold/entrypoint.go

```diff
@@ -163,7 +171,13 @@ func LinkDisks(diskKind string, num int) (err error) {
 
 func executeVirtV2v(args []string) (err error) {
 	virtV2vCmd := exec.Command(args[0], args[1:]...)
-	virtV2vCmd.Stdout = os.Stdout
+	virtV2vStdoutPipe, err := virtV2vCmd.StdoutPipe()
+	if err != nil {
+		fmt.Printf("Error setting up stdout pipe: %v\n", err)
+		return
+	}
+
+	tee := io.TeeReader(virtV2vStdoutPipe, os.Stdout)
```

Comment:
Fixed the monitor consuming issue, now the conversion phase work properly.

Reply:
![Screenshot from 2024-02-18 15-18-57](https://github.com/kubev2v/forklift/assets/73168733/d54a125e-63e9-4093-95e0-d0ea8fad6222)